src/mcts/search.h

```diff
@@ -196,6 +197,22 @@ class Search {
   friend class SearchWorker;
 };
 
+// Helper to score and sort terminals for preferred moves.
+class Search::TerminalScore {
+ public:
+  EdgeAndNode edge;
```

Comment:
A public data member prevents encapsulation and should be private: <https://google.github.io/styleguide/cppguide.html#Access_Control>

Reply:
Removed public data member